### PR TITLE
docs: add installation steps

### PR DESCRIPTION
**Pull request by rayacers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+1. Run yarn to install the dependencies.
+2. Run the API at `localhost:3000`
+3. Set the following environnement variables in `.env.local`
+
+```
+REACT_APP_API_HOST=http://localhost:3000
+PORT=3113
+REACT_APP_SHOW_NOTIFICATIONS=true
+REACT_APP_AUTHENTICATION_HOST=http://localhost:3001
+REACT_APP_GRAASP_PERFORM_HOST = http://localhost:3112
```
Comment (spaenleh) [inline README.md:16]:
```suggestion
REACT_APP_GRAASP_PERFORM_HOST=http://localhost:3112
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+REACT_APP_HIDDEN_ITEM_TAG_ID=b5373e38-e89b-4dc7-b4b9-fd3601504467
```
Comment (spaenleh) [inline README.md:17]: Maybe add a comment that this should match the hidden tag id variable in the backend env file.

File: README.md [hunk 3]
```diff
@@ -2,6 +2,28 @@
...
+REACT_APP_GRAASP_ANALYZER_HOST=http://localhost:3113
+REACT_APP_DOMAIN=localhost
+REACT_APP_H5P_ASSETS_BASE_URL=https://h5p.dev.graasp.org/h5p-assets
+REACT_APP_H5P_CONTENT_BASE_URL=https://h5p.dev.graasp.org/h5p-content
+REACT_APP_ENABLE_MOCK_API=false
+```
+
+4. Run yarn start. The client should be accessible at `localhost:3113`
```
Comment (spaenleh) [inline README.md:25]:
```suggestion
4. Run `yarn start`. The client should be accessible at `localhost:3113`
```

File: README.md [hunk 4]
```diff
@@ -2,6 +2,28 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Installation
+
+1. Run yarn to install the dependencies.
```
Comment (spaenleh) [inline README.md:7]:
```suggestion
1. Run `yarn` to install the dependencies.
```

File: README.md [hunk 5]
```diff
@@ -2,6 +2,28 @@
...
+2. Run the API at `localhost:3000`
```
Comment (spaenleh) [inline README.md:8]: Maybe here add a link to the graasp/graasp repo.
```suggestion
2. Run [the API](https://github.com/graasp/graasp) at `localhost:3000`
```

File: README.md [hunk 6]
```diff
@@ -2,6 +2,28 @@
...
+3. Set the following environnement variables in `.env.local`
```
Comment (spaenleh) [inline README.md:9]:
```suggestion
3. Set the following environnement variables in `.env.local` (place this file at your project root)
```

File: README.md [hunk 7]
```diff
@@ -2,6 +2,28 @@
...
+REACT_APP_GRAASP_ANALYZER_HOST=http://localhost:3113
+REACT_APP_DOMAIN=localhost
+REACT_APP_H5P_ASSETS_BASE_URL=https://h5p.dev.graasp.org/h5p-assets
+REACT_APP_H5P_CONTENT_BASE_URL=https://h5p.dev.graasp.org/h5p-content
+REACT_APP_ENABLE_MOCK_API=false
+```
+
+4. Run yarn start. The client should be accessible at `localhost:3113`
+
 ## Available Scripts
```
Comment (spaenleh) [inline README.md:27]: Remove this section. It is coming from the CRA template and is not very useful in this context.

File: README.md [hunk 8]
```diff
@@ -68,3 +90,7 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/de
 ### `yarn build` fails to minify
 
 This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+
+## Screenshot
+
+<img width="1381" alt="Screenshot 2023-04-27 at 4 58 43 PM" src="https://user-images.githubusercontent.com/13879502/234903339-a95ee2b8-a89e-4edb-a22a-20e9397346a6.png">
```
Comment (spaenleh) [inline README.md:96]: Maybe put this image also at the top, so it looks a bit more interesting when you arrive on the repo 😄

**Review by spaenleh — CHANGES_REQUESTED** [review]
looks good, thank you for the documentation ! Much need 😅  Please have a look at my comments, integrate the ones that make sense to you.

**Review by pyphilia — APPROVED** [review]
Thanks for the update! 🦓

**Review by spaenleh — APPROVED** [review]
👍

**Pull request merged**